consensus/state_processing/src/per_block_processing/errors.rs

```diff
@@ -89,6 +89,46 @@ pub enum BlockProcessingError {
         found: Hash256,
     },
     WithdrawalCredentialsInvalid,
+    TooManyPendingConsolidations {
+        consolidations: usize,
+        limit: usize,
+    },
+    ConsolidationChurnLimitTooLow {
+        churn_limit: u64,
+        minimum: u64,
+    },
+    MatchingSourceTargetConsolidation {
+        index: u64,
+    },
+    InactiveConsolidationSource {
+        index: u64,
+        current_epoch: Epoch,
```

Comment:
Is this error property useful? This error will appear in the context of a block at slot N with root X. It's trivial to get the epoch from there

Reply:
this error is actually not necessary in the latest spec, mark's already got it removed here: https://github.com/sigp/lighthouse/pull/5994/files

so I'll just leave it